docs/blitz/design/actions/yang.rst

```diff
@@ -179,8 +181,8 @@ use of badnwidth.  The device must be able to decode the Base64 val if this para
 
 .. _val: https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#231-json-and-json_ietf
 
-sample_interval
-~~~~~~~~~~~~~~~
+sample_poll
+~~~~~~~~~~~
```

Comment:
You changed the name to ``sample_poll`` but you did not change the comment in the section.  There is nothing explaining the ``_poll``.

Reply:
I don't think there need to be any updates in this description. Old description of sample_interval is good enough and applies to both modes

Reply:
The description only contains sample and stream. The description should be more detailed than the comment.  Please describe the use of the number for STREAM and SAMPLING and describe the use of the number for POLL.  The documentation should leave no room for misinterpretation.